rana_qgis_plugin/libs/bridgestyle/mapboxgl/fromgeostyler.py

```diff
@@ -71,9 +71,10 @@ def toSpriteSheet(allSprites):
     img, img2x, painter, painter2x, spritesheet, spritesheet2x = qgis2geostyler.initSpriteSheet(width, height)
     x = 0
     for name, _sprites in allSprites.items():
+        name_without_ext = os.path.splitext(name)[0]
```

Comment:
I think this solves the issue, however there should be logic somewhere in bridge-style that creates the sprite name in this way already.

This because the sprite name (without extension) is already used in the style json (in the symbolizers). I can't find where this happens, however. I think we should find it to make sure the logic is exactly the same.